ibis/expr/types/relations.py

```diff
@@ -675,7 +710,7 @@ def _ensure_expr(self, expr):
 
     @property
     def columns(self) -> list[str]:
-        """The list of columns in this table.
+        """The `list[str]` of column names in this table.
```

Comment:
Can we keep this docstring as is? The type annotation is already there, and the docstring is about the concept, not the specific way in which it is implemented.

Reply:
Do you see how in the live docs on the website you can't see the type annotation?

Reply:
filed at https://github.com/machow/quartodoc/issues/284